x-pack/plugins/ml/public/privilege/common.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Dictionary } from '../../common/types/common';
```

Comment:
I don't think `Dictionary` is needed here, these could be written using `Record`.
`export type Cluster = Record<string, boolean>`

Reply:
👍 Fixed in f346a4f.